app/src/test/java/net/squanchy/support/ZonedDateTime.kt

```diff
@@ -0,0 +1,9 @@
+package net.squanchy.support
+
+import org.threeten.bp.DateTimeUtils
+import org.threeten.bp.ZonedDateTime
+import java.util.Date
+
+fun ZonedDateTime.toDate(): Date {
```

Comment:
Could be inlined as 
`fun ZonedDateTime.toDate(): Date = DateTimeUtils.toDate(toInstant())`

Reply:
Also, nit, this seems to only be used in one place; can it be a private extension fun on that file instead?

Reply:
it's used multiple times (24, it's in the tests), :+1: for inlining it

Reply:
e68d74b

Reply:
Textual search on this PR only gave me one 😓 I failed